python/artexplainer.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7
+FROM python:3.9-slim-bullseye
```

Comment:
switched to using `python:3.9-slim-bullseye` instead of `python:3.9-slim` as building with this results to 3 less critical CVEs being reported:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/35975404/175496713-6c2c39cb-76ce-4d3a-9f37-d547b04b87c4.png">
<img width="680" alt="image" src="https://user-images.githubusercontent.com/35975404/175496801-525c606d-b54c-49c9-abba-7b8b3fe5ced0.png">